### PR TITLE
Add conda binary to $PATH first before initialize.

### PR DESCRIPTION
**Pull request by mpotane**
### Description
Cannot invoke conda init fish because fish doesn't auto add the binary to $PATH #8590

### Checklist - did you ...

- [x] Add / update documentation?

### CODE REVIEW

**Comment by mpotane** [comment]
> We require contributors to sign our [Contributor License Agreement](https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement) and we don't have one on file for @mpotane.
> 
> In order for us to review and merge your code, please e-sign the [Contributor License Agreement PDF](https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement). We then need to manually verify your signature. We will ping the bot to refresh the PR status when we have confirmed your signature.

done signing 😁

File: docs/source/user-guide/install/linux.rst [hunk 1]
```diff
@@ -44,7 +44,13 @@ Using with fish shell
 
 To use conda with fish shell, run the following in your terminal:
 
-  ``conda init fish``
+ #. Add conda binary to $PATH, if not yet added::
+ 
+      fish_add_path path/to/bin
```
Comment (jaimergp) [inline docs/source/user-guide/install/linux.rst:49]:
```suggestion
      fish_add_path path/to/condabin
```

It's best to add only the `condabin` directory, which is designed to contain only conda binaries, and not other stuff that might be installed in `base` (e.g. `python`).

**Comment by jezdez** [comment]
@conda-bot check

**Comment by mpotane** [comment]
pre-commit.ci autofix

File: docs/source/user-guide/install/linux.rst [hunk 2]
```diff
@@ -44,7 +44,13 @@ Using with fish shell
 
 To use conda with fish shell, run the following in your terminal:
 
-  ``conda init fish``
+ #. Add conda binary to $PATH, if not yet added::
+
+      fish_add_path path/to/condabin
```
Comment (travishathaway) [inline docs/source/user-guide/install/linux.rst:49]:
```suggestion
      fish_add_path <conda-install-location>/condabin
```

I think this makes it a little more clear that the `condabin` folder is located where conda itself was installed.

**Review by travishathaway — CHANGES_REQUESTED** [review]
Hi @mpotane,

Sorry for the late turn around on this 🙃. I made one suggestion to your documentation changes. I think if we incorporate that, this pull request will be ready to merge.

**Review by travishathaway — APPROVED** [review]

**Comment by jezdez** [comment]
@travishathaway @mpotane https://github.com/conda/conda/pull/12070#issuecomment-1578954925 applies here, `fish_add_path` is a feature that was only added in fish 3.2.0 which may not be available in all Linux distributions yet. This is related to the unresolved #11079 as well.

**Pull request merged**